sdk/identity/identity/managed-identity-matrix.json

```diff
@@ -0,0 +1,19 @@
+{
+  "displayNames": {
+    "**/test-results.xml": ""
+  },
+  "include": [
+    {
+      "Agent": {
+        "msi_image": {
+          "OSVmImage": "env:LINUXNEXTVMIMAGE",
+          "Pool": "env:LINUXNEXTPOOL"
+        }
+      },
+      "ArmTemplateParameters": "@{deployMIResources = $true}",
+      "TestType": "msi",
+      "NodeTestVersion": "18.x",
```

Comment:
nit: Node 20?

Reply:
I guess we did 18 everywhere. Isn't that the LTS?